extensions/service_profiles/standard/requirements/core/REQ_pubsub.adoc

```diff
@@ -0,0 +1,14 @@
+[[req_service-profile_pubsub]]
+
+[requirement]
+====
+[%metadata]
+identifier:: /req/service-profile/pubsub
+
+part:: A requirement MAY be defined to which specifies support for EDR Part 2 pubsub.
```

Comment:
Please use correct name: `OGC API-EDR Part 2: Publiosh-Subscribe Workflow`.

Reply:
OGC API - Environmental Data Retrieval - Part 2: Publish-Subscribe workflow.

Reply:
Added correct name for Part 2 to the requirement.